os_builders/roles/vm_baseline/tasks/pakiti.yml

```diff
@@ -46,4 +46,21 @@
     name: "/tmp/pakiti-client-2.1.4-3.RAL.noarch.rpm"
     state: present
     disable_gpg_check: true
-  when: ansible_distribution == "Rocky"
+  when: ansible_distribution == "Rocky"  and "ccm" not in ansible_facts.packages
```

Comment:
With these, wouldn't Aquilon simply remove the packages?
I'm tempted to ship the kitchen-sink so we have the same set across all images then simply let the config management tool remove anything extra later

Reply:
As these get run against already managed images as part of the packuilon build I'd rather not have them fighting during the build